hc/accounts/urls.py

```diff
@@ -25,4 +25,4 @@
         views.switch_team, name="hc-switch-team"),
 
 
-]
+]
```

Comment:
Add a new line at the end of this file. Get an extension for this for your editor.